news/2019-06-12-project-loved-week-of-june-9th.md

```diff
@@ -0,0 +1,226 @@
+
+## <a name="osucatch" id="osucatch"></a>osu!catch
+
+osu!catch Loved candidates were chosen by [Sartan](https://osu.ppy.sh/users/4100941), [Tenshichan](https://osu.ppy.sh/users/1101600), [Wesley](https://osu.ppy.sh/users/2407265) and [Yumeno Himiko](https://osu.ppy.sh/users/1806962)!
+
+---
+
+[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/catch/1-augoeides.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[DystopiaGround - AugoEidEs](https://osu.ppy.sh/beatmapsets/191008#fruits) by [Serena-](https://osu.ppy.sh/users/2193444), [Minato Yukina](https://osu.ppy.sh/users/531253), [ExGon](https://osu.ppy.sh/users/214187), [Frobe](https://osu.ppy.sh/users/670365), [Last Bubble](https://osu.ppy.sh/users/1302259) and [Spectator](https://osu.ppy.sh/users/702598)  
+207 BPM, 6:51 | 6.08★, 6.14★  
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+AugoEidEs is a song most osu!catch players will know very well, but this time the map is created by several Korean players. The mapset has two fully-mapped difficulties. "Crystal" is made by Minato Yukina, while "Darkness" is a collaboration by Serena-, ExGon, Frobe, Minato Yukina and Last Bubble where Spectator handled the hitsounds. Both difficulties are what you would expect of such an intense song: a lot of streams, jumps and of course a lot of creativity. As the map is somewhat old already (2014), it's a lot different compared to the newer AugoEidEs mapped by Koliron. Expect to see a lot of "older" stream patterns and jumps while playing either of these two difficulties!
+
+[![](/wiki/shared/news/2019-06-12-project-loved-week-of-june-9th/catch/2-all-i-need-are-things-i-like.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+[nameless - All I Need are Things I Like](https://osu.ppy.sh/beatmapsets/742694#fruits) by [Alostis](https://osu.ppy.sh/users/2986875)  
+140 BPM, 4:17 | 7.17★  
+*description written by [Sartan](https://osu.ppy.sh/users/4100941)*
+
+Another round, another first timer for the osu!catch Loved vote! Here we have Alostis with his most popular map, All I Need are Things I Like. Despite the very small quantity of maps he has created, Alostis has put out a true banger with this second map of his. For its time of release last year, this map has been able to accumulate quite a respectable amount of favourites from players, and with good reason. The combination of the cute and funky song, together with very enjoyable jumping patterns for players on a pretty high level has made this map quite a success, with it appearing in multiplayer lobbies quite often. Mods can also very well be applied to this map, with no patterns becoming clunky with HardRock and Hidden being quite friendly thanks to the pretty high AR of 9.6. So go check it out, if you need new things to like!
```

Comment:
```suggestion
Another round, another first timer for the osu!catch Loved vote! Here we have Alostis with his most popular map, All I Need are Things I Like. Despite the very small quantity of maps he has created, Alostis has put out a true banger with this second map of his. Since its time of release last year, this map has been able to accumulate quite a respectable amount of favourites from players, and for good reason. The combination of the cute and funky song, together with very enjoyable jumping patterns for players on a pretty high level has made this map quite a success, with it appearing in multiplayer lobbies quite often. Mods can also very well be applied to this map, with no patterns becoming clunky with HardRock and Hidden being quite friendly thanks to the pretty high AR of 9.6. So go check it out, if you need new things to like!
```
grammar